python/cuda_parallel/cuda/parallel/experimental/_bindings.pyx

```diff
@@ -0,0 +1,1620 @@
+            prop_name,
+            cccl_type_enum.CCCL_FLOAT64
+        )
+
+
+    cdef IntEnumerationMember make_STORAGE(self):
+        cdef str prop_name = "STORAGE"
+        return IntEnumerationMember(
+            type(self),
+            self.enum_name,
+            prop_name,
+            cccl_type_enum.CCCL_STORAGE
+        )
+
+
+cdef class Enumeration_OpKind(IntEnumerationBase):
+    "Enumeration of operator kinds"
+    cdef IntEnumerationMember _stateless
+    cdef IntEnumerationMember _stateful
+
+    def __cinit__(self):
+        self.enum_name = "OpKindEnum"
+        self._stateless = self.make_STATELESS()
+        self._stateful = self.make_STATEFUL()
+
+    cdef IntEnumerationMember make_STATELESS(self):
+        cdef str prop_name = "STATELESS"
+        return IntEnumerationMember(
+            type(self),
+            self.enum_name,
+            prop_name,
+            cccl_op_kind_t.CCCL_STATELESS
+        )
+
+    cdef IntEnumerationMember make_STATEFUL(self):
+        cdef str prop_name = "STATEFUL"
+        return IntEnumerationMember(
+            type(self),
+            self.enum_name,
+            prop_name,
+            cccl_op_kind_t.CCCL_STATEFUL
+        )
+
+
+    @property
+    def STATELESS(self):
+        return self._stateless
+
+    @property
+    def STATEFUL(self):
+        return self._stateful
+
+
+cdef class Enumeration_IteratorKind(IntEnumerationBase):
+    "Enumeration of iterator kinds"
+    cdef IntEnumerationMember _pointer
+    cdef IntEnumerationMember _iterator
+
+    def __cinit__(self):
+        self.enum_name = "IteratorKindEnum"
+        self._pointer = self.make_POINTER()
+        self._iterator = self.make_ITERATOR()
+
+    cdef IntEnumerationMember make_POINTER(self):
+        cdef str prop_name = "POINTER"
+        return IntEnumerationMember(
+            type(self),
+            self.enum_name,
+            prop_name,
+            cccl_iterator_kind_t.CCCL_POINTER
+        )
+
+    cdef IntEnumerationMember make_ITERATOR(self):
+        cdef str prop_name = "ITERATOR"
+        return IntEnumerationMember(
+            type(self),
+            self.enum_name,
+            prop_name,
+            cccl_iterator_kind_t.CCCL_ITERATOR
+        )
+
+    @property
+    def POINTER(self):
+        return self._pointer
+
+    @property
+    def ITERATOR(self):
+        return self._iterator
+
+
+TypeEnum = Enumeration_CCCLType()
+OpKind = Enumeration_OpKind()
+IteratorKind = Enumeration_IteratorKind()
+
+
+cpdef bint is_TypeEnum(IntEnumerationMember attr):
+    "Return True is attribute represents a type enumerator"
+    return attr.parent_class is Enumeration_CCCLType
+
+
+cpdef bint is_OpKind(IntEnumerationMember attr):
+    "Return True is attribute represents an enumerator of operator kinds"
+    return attr.parent_class is Enumeration_OpKind
+
+
+cpdef bint is_IteratorKind(IntEnumerationMember attr):
+    "Return True is attribute represents an enumerator of iterator kinds"
```

Comment:
Minor thing but is this supposed to be `if attribute` instead of `is attribute`? Same for `OpKind` and `TypeEnum` above

Reply:
Fixed docstrings.